solidity/libraries/MerklePatriciaProofVerifier.sol

```diff
@@ -51,12 +51,12 @@ library MerklePatriciaProofVerifier {
 
       // The root node is hashed with Keccak-256 ...
       if (i == 0 && rootHash != stack[i].rlpBytesKeccak256()) {
-        revert();
+        revert('revert');
```

Comment:
let's use some better message for the reverts

Reply:
These are from the lido library and they have them as unnamed i added these when debugging, hut i think we should go back to how they have it